econml/inference/_inference.py

```diff
@@ -425,9 +431,12 @@ def effect_inference(self, X, *, T0, T1):
         pred = pred[np.arange(T0.shape[0]), ..., ind]
         pred_stderr = pred_stderr[np.arange(T0.shape[0]), ..., ind] if pred_stderr is not None else None
         # d_t=1 here since we measure the effect across all Ts
+        # A dummy treatment names is passed in
```

Comment:
Consider rewording, since it is not actually passed in

```suggestion
        # Omit treatment name argument to fall back to the default
```